### ISSUE TITLE
Personal issue ;)

### ISSUE THREAD

**Issue by bouncyball-git** [ilia3101/MLV-App#87]
@ilia3101 @masc4ii 

Hey guys I'm somehow is out of contributors list?!
Do you know what is the reason of this?

**Comment by ilia3101** [comment]
@bouncyball-git I noticed this a few days ago and was really confused, as all of the commits by you no longer linked to a profile, also saw that you weren't on the contributors list. Thought you had left github, but then I realised your profile was still there. I have no idea how this happened. It's sad that github isn't showing your contribution, hope it comes back. See what happens if you send a commit, will it say you have contributed just that? or maybe would fix itself somehow idk

Sorry I have been away from MLV App for a little. Have come up with some ideas for sorting out whitebalance branch. Also working on something else but not got very far on that yet (it could be helpful for this). Had a lot of other things to do.

**Comment by masc4ii** [comment]
@bouncyball-git : did you change your account name maybe? "bouncyball" vs. "bouncyball-git"? 
Sorry to be also not with you for some days / weeks. I am in holiday and my internet is very limited here... same for my time here - I really must enjoy that - dreaming my whole life of coming here. Therefor I'll come back with some nice new MLVs and don't have to test everything with the well known boat clip 😄 (after 4 days I have already about 300GB)
Greetings from the Seychelles!

**Comment by bouncyball-git** [comment]
Hehe! Hi guys nice to talk to both of you. I just changed email address (not the name) of my github account b/c old address will be shut down after a month by my mail provider and I substituted it with my other gmail address. This could be the issue, maybe.

@ilia3101: nice to hear you have some goodies for us ;)
@masc4ii : have a great time man! Awaiting new test MLVs ;)

Unfortunately I'm very busy these days too...
